clog/context.go

```diff
@@ -78,3 +80,13 @@ func ContextWithCloudType(ctx context.Context, key string) context.Context {
 func ContextWithSchedulerType(ctx context.Context, key string) context.Context {
 	return context.WithValue(ctx, contextSchedulerType, key)
 }
+
+// ContextWithCallValue returns a context which knows the given call value.
+func ContextWithCallValue(ctx context.Context, key string) context.Context {
+	return context.WithValue(ctx, contextCallValue, key)
+}
+
+// ContextWithServerFlavour returns a context which knows the given server flavour.
```

Comment:
Just do a full search and replace for Flavour -> Flavor

Reply:
Ah docs!!! 
fixed now.